api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java

```diff
@@ -15,6 +15,7 @@
 import org.hibernate.SessionFactory;
 import org.openmrs.Condition;
 import org.openmrs.Patient;
+import org.openmrs.api.ConditionService;
```

Comment:
do we need this import here ??

Reply:
As you can see, there was a line in the original code : 
/**
* @see ConditionService#getAllConditions(Patient)
 */
To properly reference, ConditionService must be imported!

Reply:
Do you get a compiler error when you remove the import?

Reply:
@dkayiwa ,that import is only used by javadoc tags

Reply:
Do we need it if we do not get a compiler error?

Reply:
Removed the import.